xml/cleanup_and_order_mads.xsl

```diff
@@ -0,0 +1,175 @@
+    <xsl:apply-templates select="mads:titleInfo"/>
+    <xsl:apply-templates select="mads:topic"/> 
+    <xsl:apply-templates select="mads:temporal"/> 
+    <xsl:apply-templates select="mads:genre"/>
+    <xsl:apply-templates select="mads:geographic"/>
+    <xsl:apply-templates select="mads:hierarchicalGeographic"/> 
+    <xsl:apply-templates select="mads:occupation"/>
+    </xsl:copy>
+  </xsl:template>
+  
+
+  <!-- Reorder <mads:name> child elements -->
+  <xsl:template match="mads:name">
+    <xsl:copy>
+      <xsl:apply-templates select="mads:namePart"/>
+      <xsl:apply-templates select="mads:description"/>
+    </xsl:copy>
+  </xsl:template>
+    
+  <!-- Reorder <mads:affiliation> child elements -->
+  <xsl:template match="mads:affiliation">
+    <xsl:copy>
+      <xsl:apply-templates select="mads:position"/>
+      <xsl:apply-templates select="mads:organization"/>
+      <xsl:apply-templates select="mads:address"/> 
+      <xsl:apply-templates select="mads:email"/> 
+      <xsl:apply-templates select="mads:phone"/>
+      <xsl:apply-templates select="mads:fax"/>
+      <xsl:apply-templates select="mads:hours"/> 
+      <xsl:apply-templates select="mads:dateValid"/> 
+    </xsl:copy>
+  </xsl:template>
+  
+
+  <!-- Reorder <mads:titleInfo> child elements -->
+  <xsl:template match="mads:titleInfo">
+    <xsl:copy>
+      <xsl:apply-templates select="mads:nonSort"/>
+      <xsl:apply-templates select="mads:title"/>
+      <xsl:apply-templates select="mads:subTitle"/>
+      <xsl:apply-templates select="mads:partNumber"/>
+      <xsl:apply-templates select="mads:partName"/>
+     </xsl:copy>
+  </xsl:template>
+  
+  
+  <!-- Reorder <mads:hierarchicalGeographic> child elements -->
+  <xsl:template match="mads:hierarchicalGeographic">
+    <xsl:copy>
+      <xsl:apply-templates select="mads:continent"/>
+      <xsl:apply-templates select="mads:country"/>
+      <xsl:apply-templates select="mads:province"/>
+      <xsl:apply-templates select="mads:region"/>
+      <xsl:apply-templates select="mads:state"/>
+      <xsl:apply-templates select="mads:territory"/>
+      <xsl:apply-templates select="mads:county"/>
+      <xsl:apply-templates select="mads:city"/>
+      <xsl:apply-templates select="mads:island"/>
+      <xsl:apply-templates select="mads:area"/>
+      <xsl:apply-templates select="mads:extraterrestrialArea"/>
+      <xsl:apply-templates select="mads:citySection"/>
+    </xsl:copy>
+  </xsl:template>
+    
+  <!-- Reorder <mads:address> child elements -->
+  <xsl:template match="mads:address">
+    <xsl:copy>
+      <xsl:apply-templates select="mads:street"/>
+      <xsl:apply-templates select="mads:city"/>
+      <xsl:apply-templates select="mads:state"/> 
+      <xsl:apply-templates select="mads:country"/> 
+      <xsl:apply-templates select="mads:postcode"/>
+    </xsl:copy>
+  </xsl:template>
+  
+  <!-- Reorder <mads:language> child elements -->
+  <xsl:template match="mads:langauge">

```

Comment:
thanks for reviewing. I'll fix the typo and do another pull request.
dm

>>> Jared Whiklo <notifications@github.com> 06/10/15 12:47 PM >>>
> +    </xsl:copy>
> +  </xsl:template>
> +    
> +  <!-- Reorder <mads:address> child elements -->
> +  <xsl:template match="mads:address">
> +    <xsl:copy>
> +      <xsl:apply-templates select="mads:street"/>
> +      <xsl:apply-templates select="mads:city"/>
> +      <xsl:apply-templates select="mads:state"/> 
> +      <xsl:apply-templates select="mads:country"/> 
> +      <xsl:apply-templates select="mads:postcode"/>
> +    </xsl:copy>
> +  </xsl:template>
> +  
> +  <!-- Reorder <mads:language> child elements -->
> +  <xsl:template match="mads:langauge">

Typo "mads:language"

---
Reply to this email directly or view it on GitHub:
https://github.com/Islandora/islandora_solution_pack_entities/pull/77/files#r32131892